app/js/controllers/main.js

```diff
@@ -190,4 +182,21 @@ export default class MainController extends Controller {
 
     this._isOpen = false;
   }
+
+  openAddonManager() {
+    var activity = new MozActivity({
+      name: 'configure',
+      data: {
+        target: 'device',
+        section: 'addons',
+        options: {
+          manifestURL: this.manifestURL
```

Comment:
We should tell @yzen about this. I would suggest a more specific parameter name. This is too vague, as it could be referring to an add-on's manifest URL. Maybe ```filterManifestURL``` or ```targetAppManifestURL```.